src/Mono.Android/Xamarin.Android.Net/ServerCertificateCustomValidator.cs

```diff
@@ -158,33 +159,29 @@ private sealed class AlwaysAcceptingHostnameVerifier : Java.Lang.Object, IHostna
 			public bool Verify (string? hostname, ISSLSession? session) => true;
 		}
 
-		private static IX509TrustManager? FindX509TrustManager(ITrustManager[]? trustManagers)
+		[DynamicDependency(nameof(IX509TrustManager.CheckServerTrusted), typeof(IX509TrustManagerInvoker))]
+		[DynamicDependency(nameof(IX509TrustManager.CheckServerTrusted), typeof(X509ExtendedTrustManagerInvoker))]
```

Comment:
Where is the use of these dependencies - I assume we have a suppression for it.
I'm just curious why we can't solve this any other way...